src/services/ApiService.tsx

```diff
@@ -0,0 +1,47 @@
+class ApiService 
+{
+  // Accept api endpoint and body parameters to make a POST request
+  static async postRequest<T>(url: string, requestBody: any): Promise<T> 
+  {
+    try
+    {
+      const response = await fetch(url, {
```

Comment:
Initialize base url based on the environment `dev/stage/prod` and append path passed in parameter to construct full url